HotelPriceScout/Pages/Component.razor

```diff
@@ -0,0 +1,48 @@
+﻿@page "/vaterpas"
+<h3>Price thermometer</h3>
+
+<div class="row border col-md-4"> <!-- outer layer-->
+
+    @if(Test == null){<div><p>Loading data</p> </div>}// checks if thermometer is ready yet
+    else{
+
+        @foreach (Prices item in Test)
+            {
+                @if(item.Price > MarketPriceItem.Price) 
+                {
+                    <div class='row col-md-12 align-items-center'><div class='col text-right' id='left-name'><p>@item.Name <span class="oi oi-arrow-right"></span></p></div>
+                    <div class='col-md-3 bg-danger '><p>@item.Price,-</p></div>
+                    @*<div class='col text-left'><p>Blank Name</p></div>*@</div>
+                } 
+                else if (item.Price < MarketPriceItem.Price)
+                {
+                   <div class="row col-md-12 align-items-center"><div class="col text-right" id="left-name"><p class="align-middle">@item.Name <span class="oi oi-arrow-right"></span></p></div>
+                   <div class="col-md-3 bg-success"><p class="align-middle">@item.Price,-</p></div>
+                   @*<div class="col text-left"><p>Blank Name</p></div>*@</div>
+                }else 
+                {
+                    <div class='row col-md-12 align-items-center'><div class='col text-right' id='left-name'><p>@MarketPriceItem.Name <span class="oi oi-arrow-right"></span></p></div>
+                   <div class='col-md-3 bg-primary'><p>@MarketPriceItem.Price,-</p></div>
+                   @*<div class='col text-left'><p>Blank Name</p></div>*@</div>
+                }
+                
+            }      
+    }
+ </div>
+
+
+@code 
+{
+    protected override async Task OnInitializedAsync() 
+    { generatetermometer(); }
+
+    List<Prices> Test;
+    Prices MarketPriceItem;
+    private void generatetermometer()
```

Comment:
Vi skal sørge for at holde os til C# standarder. Alle methods skal gerne være med stort forbogstav på alle ord. Så GenerateThermometer() ville være rigtigt.